src/main.cpp

```diff
@@ -18,6 +18,12 @@
 
 #include <stdio.h>
 
+// Define this symbol here so that we can link with the zig objects during the
+// compiler bootstrap phase
+extern "C" {
+    void __zig_probe_stack(void) { }
```

Comment:
@LemonBoy why was this needed? I tried removing this and it seems to work fine

Reply:
I needed that in order not to do a fresh rebuild when I first added the stack probe annotation, you can safely remove it now.